config/ember-try.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
 
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
+      embroiderSafe(),
+      embroiderOptimized(),
       {
         name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
             'ember-source': '~3.28.0',
             'ember-qunit': '^5.0.0',
             '@ember/test-helpers': '^2.4.0',
-            '@glimmer/component': '^1.0.0',
+            '@glimmer/component': '^1.1.2',
```

Comment:
Per the glimmer/component changelog, v2 only supports ember 4.10+, so we downgrade to glimmer@1 for the ember-try scenarios with older ember.

https://github.com/emberjs/ember.js/blob/main/packages/%40glimmer/component/CHANGELOG.md